packages/payments/src/constants/settings.ts

```diff
@@ -0,0 +1,31 @@
+const isDev = (): boolean => {
+  return !!(
+    typeof window !== "undefined" &&
+    window.localStorage.getItem("IS_PAPER_DEV") === "true"
+  );
+};
+
+const isOldPaperDomain = (): boolean =>
+  typeof window !== "undefined" && window.location.origin.includes("paper.xyz");
```

Comment:
check if this is still relevant somehow?

Reply:
this is bc some things are used by paper.xyz vs withpaper.com

Reply:
fwiw this origin check is bad, it should instead check if the origin `endsWith(".paper.xyz")` (assuming the origin is expected to be `www.paper.xyz`)